config/routes.rb

```diff
@@ -11,6 +11,7 @@
   resources :companies, only: %i(index show)
   resources :users do
     resources :practices, only: %i(index), controller: 'users/practices'
+    resources :reports, only: %i(index), controller: 'users/reports'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.